services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    new_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return new_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
```

Comment:
is session_date a str?

Reply:
It should be a str.
- `get_movies_sessions`, takes optional string `session_date` in such
form: "year-month-day" - this is from readme file